### ISSUE TITLE
Возможность предварительного просмотра резюме

### ISSUE THREAD

**Issue by TarakanovAndrey** [Hexlet/hexlet-cv#694]
### Problem description

В форме резюме есть поля, которые поддерживают markdown. Очень удобно иметь возможность увидеть промежуточный результат перед тем как опубликовать или сохранить резюме в черновик.

### Proposed solution

Внедирть предварительный просмотр отедльных полей или всего резюме в целом

**Comment by fey** [comment]
Как вариант, у нас есть черновик резюме. В теории если резюме в таком статусе, то можно разрешить автору его просматривать.

**Comment by fey** [comment]
Можно брать в работу.

**Comment by fey** [comment]
@beherit197777

**Comment by beherit197777** [comment]
беру в работу

**Comment by beherit197777** [comment]
https://github.com/fey  я все сделал, надо на рендер заливать ?

**Comment by fey** [comment]
> я все сделал, надо на рендер заливать ?

@beherit197777 да, можно. Только сперва пулл реквест создайте) и в нем можете ссылку дать на демо.
